docs/usermanual-integration.xml

```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+               "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd" [
+  <!ENTITY % local.common.attrib "xmlns:xi  CDATA  #FIXED 'http://www.w3.org/2003/XInclude'">
+  <!ENTITY version SYSTEM "version.xml">
+]>
+<chapter id="integration">
+  <title>Platform Integration Guide</title>
+  <para>
+    HarfBuzz was first developed for use with the GNOME and GTK
```

Comment:
Well, the intent of this sentence is to explain (a) why there are extensive GTK / Pango -isms in the code (starting with gtkdoc markup elsewhere in the docs, for example...) while also (b) reassuring developers that these isms are not a limitation so they should forge ahead anyway. Would losing the "first" resolve the issue?

Reply:
The only GTK-ism we have is the use of gtkdoc and gir-introspection, which does not affect users of the library in anyway. But if @behdad is fine with this sentence, I have no objection.

Reply:
There is also GLib stuff.

Reply:
Yes, but we have ICU and FreeType stiff as well. My main objection is that GTK and Pango have many negative associations these days, and I don’t want it to look like HarfBuzz is a GNOME project.

Reply:
I think we're in total agreement as to the goal. I could use some clarity on if there's a change in the wording that would make it meet the goal.

My belief would be that not 'mentioning GTK/GNOME at all' doesn't help achieve the goal, since many people just discovering HB are aware that there's _some_ connection already. So I am trying to figure out what to say that acknowledges the connection, and yet puts it into the proper context so as to encourage people on other platforms to forge ahead. Maybe something that gets tweaked several times before it is perfect; I don't want it to be a blocker.